frontend_lib/src/helper.js

```diff
@@ -394,3 +395,52 @@ export const getCurrentContentVersionNumber = (appFeatureMode, content, timeline
   if (appFeatureMode === APP_FEATURE_MODE.REVISION) return content.number
   return timeline.filter(t => t.timelineType === 'revision' && t.hasBeenRead).length
 }
+
+export const wrapMentionInSpanTag = (text) => {
+  try {
+    const textArray = text.split('&nbsp;').join(' ').split(' ')
```

Comment:
```suggestion
    const textArray = text.split(/&nbsp;| /)
```
What about \t or other white chars ? maybe use `\s` ?
With this code, you are replacing non breaking space with regular space at the end. This does not seem right.

I also think that this parsing code is a bit fragile: what happens if there is something that looks like a mention in an HTML attribute? I think you should really use a DOMParser and then loop over the text nodes.

Also, what about using a regex with a back reference to detect the space without splitting the string and potentially mangling special white spaces?
`&nbsp;` is not the only entity for spaces. 

What about `<p>@mention</p>`?

Reply:
Changed